content/CORE/GettingStarted/StoringData.md

```diff
@@ -63,4 +66,6 @@ Organize the pool with additional datasets or zvols according to your access and
 
 When you're finished building and organizing your TrueNAS pools, move on to configuring how the system [shares data]({{< relref "SharingStorage.md" >}})
 
-{{< taglist tag="corestoredata" limit="10" >}}
\ No newline at end of file
+{{< taglist tag="corestoredata" limit="10" >}}
+
+{{< taglist tag="coregettingstarted" limit="10" title="Related Articles" >}}
```

Comment:
In this second taglist shortcode suggest you change the title to this:
```suggestion
{{< taglist tag="coregettingstarted" limit="10" title="Related Getting Started Articles" >}}
```

Reply:
Completed.